build/Dockerfile

```diff
@@ -59,7 +60,11 @@ RUN --mount=type=secret,id=STEAM_USERNAME \
     i=0; \
     while [ "$i" -le $(( $INSTALL_COUNT-1 )) ]; do \
         echo "[BUILD] Install count: [$(( i+1 ))/$INSTALL_COUNT]"; \
-        steamcmd.sh $steamcmdArgs; \
+        if [ "$i" -le $(( $INSTALL_COUNT-1 )) ]; then \
+            steamcmd.sh $steamcmdArgs || true; \
+        else \
+            steamcmd.sh $steamcmdArgs; \
+        fi; \
```

Comment:
since the `Dockerfile` is short, not using `set -eu` is generally still easy to debug, but when adding more things to the `Dockerfile` thats where `set -eux` helps a lot.

Reply:
see https://github.com/startersclan/docker-sourceservers/pull/123#discussion_r1130811409